emission/analysis/intake/cleaning/filter_accuracy.py

```diff
@@ -100,7 +100,8 @@ def filter_accuracy(user_id):
                 logging.debug("Partial filtered points %d found" % len(filtered_points_df))
                 filtered_points_df = filtered_points_df[SEL_FIELDS_FOR_DUP]
                 filtered_points_df["msts"] = filtered_points_df.ts.apply(lambda x: int(x * 10**3))
-                matched_points_df = filtered_from_unfiltered_df.merge(filtered_points_df, on="msts", right_index=True)
+                matched_points_df = filtered_from_unfiltered_df.reset_index().merge(filtered_points_df, on="msts")
+                matched_points_df.set_index('index', inplace=True)
```

Comment:
I'm a little confused about this. The issue comment that you linked to says that "on" can be removed,
> Therefore on can be removed from here.

but you have actually removed the `right_index`

Reply:
@shankari 
According to the new version of Pandas, we cannot use 'left_index/right_index' and 'on' together so we can define the column on which we want to merge the dataframe and set the indexes using functions like set_index or reset_index.

In the testcase below in '[TestFilterAccuracy.py](https://github.com/e-mission/e-mission-server/pull/900/commits/a760b55cc7af16d95d6f9428a4fe0d7631b9c39d#diff-e9aaf12a6c1fcc5821ea6c95cf67a5223998ba7896dd25cfcdce1998983cd7e7)' there was only one column 'ts' therefore removing 'on' was also giving us the desired result but that won't be a good solution as we can have multiple columns in a dataframe.